fares-data-build-tool/fdbt-admin
# feature: add functionality to admin add user page and add registration counts to list user page

# Description

-   Added functionality to the admin panel add user page
-   Added completed, pending and total registration counts to the list users page (excluding IWBusCo users)

# Testing instructions

-   Clone this branch and run locally
-   Test that you can add users using the admin page and assure that the user is created in the fdbt-user-pool
-   Test that attempting to add the same user twice results in an error message
-   Test that the completed, pending and total registered user counts reflect the users displayed on the list users page (these numbers should exclude any user with IWBusCo in their NOCs list)

# Type of change


# Checklist:
